packages/indexer-common/src/errors.ts

```diff
@@ -159,6 +160,7 @@ export const INDEXER_ERROR_MESSAGES: Record<IndexerErrorCode, string> = {
   IE070: 'Failed to query latest valid epoch and block hash',
   IE071: 'Add Epoch subgraph support for non-protocol chains',
   IE072: 'Failed to execute batch tx (contract: staking)',
+  IE073: 'Failed to query subgraph features',
```

Comment:
We could probably provide a bit more detail here to help with debugging. Many indexers may not be familiar with the subgraph features query, so we could also include a reference to the indexing status endpoint on graph-node so they know more context.  

What about: 'Failed to query subgraph features from indexing statuses endpoint'?